Sources/Workspace/Workspace.swift

```diff
@@ -1589,10 +1589,8 @@ extension Workspace {
         let tempDiagnostics = DiagnosticsEngine()
         let result = ThreadSafeArrayStore<ManagedArtifact>()
 
-        var authProvider: AuthorizationProviding? = nil
-        #if os(macOS) // Netrc feature currently only supported on macOS
-        authProvider = try? Netrc.load(fromFileAtPath: netrcFilePath).get()
-        #endif
+        // FIXME: should this handle the error more gracefully?
```

Comment:
@neonichu do you know why the error is ignored here? topic for different PR, but curios